include/zephyr/bluetooth/audio/csis.h

```diff
@@ -190,16 +191,16 @@ int bt_csis_register(const struct bt_csis_register_param *param,
 void bt_csis_print_sirk(const struct bt_csis *csis);
 
 /**
- * @brief Starts advertising the Resolvable Set Identifier value.
+ * @brief Generate the Resolvable Set Identifier (RSI) value.
  *
- * This cannot be used with other connectable advertising sets.
+ * This will generate RSI for given @p csis instance.
  *
  * @param csis          Pointer to the Coordinated Set Identification Service.
- * @param enable	If true start advertising, if false stop advertising
+ * @param rsi		Pointer to the 6-octet newly generated RSI data.
```

Comment:
```suggestion
 * @param rsi           Pointer to the 6-octet newly generated RSI data.
```
spaces instead of tabs